snuba/snuba_migrations/generic_metrics/0046_distributions_add_disable_percentiles.py

```diff
@@ -0,0 +1,51 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import Column, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    blocking = False
+    local_table_name = "generic_metric_distributions_raw_local"
+    dist_table_name = "generic_metric_distributions_raw_dist"
+    storage_set_key = StorageSetKey.GENERIC_METRICS_DISTRIBUTIONS
+
+    def forwards_ops(self) -> Sequence[operations.SqlOperation]:
+        return [
+            operations.AddColumn(
+                storage_set=self.storage_set_key,
+                table_name=self.local_table_name,
+                column=Column(
+                    "disable_percentiles", UInt(8, Modifiers(default=str("0")))
```

Comment:
You can apply the T64 encoding scheme to this column since there are only ever going to be 2 values in it. That way this column will not take too much space in storage. https://clickhouse.com/docs/en/sql-reference/statements/create/table#t64